upgrade/v3-to-v4/connectors/recreate-connector-storage.sh

```diff
@@ -146,7 +149,7 @@ function getDataFromStorage(){
 
     if [[ $backupDataRequired == "true" ]]; then
 
-        targetDir=/tmp/$connconfig/$dataFilePath
+        targetDir=/tmp/$connconfig$dataFilePath
```

Comment:
Was the slash meant to be deleted?

Reply:
Yup. Because the dataFilePath variable already has a leading slash which makes it redundant.